charts/wire-server/values.yaml

```diff
@@ -9,3 +9,4 @@ tags:
   team-settings: false
   account-pages: false
   legalhold: false
+  federator: false
```

Comment:
Do we want to disable installing this compoent by default, and enable it once the implementation is more "ready"? Or do you want to keep this false also going forward?

Reply:
Yes, I was thinking we shouldn't enable it by default at least until it is useful. Even when it is useful, maybe we shouldn't deploy it when federation is not enabled.